### DISCUSSION TITLE
I suggest you add new classes for gaps

### DISCUSSION THREAD

**Discussion by qijyc101** [quasarframework/quasar#17625] [Ideas / Proposals]
.q-gap-xs {
  gap: 4px;
}
.q-gap-sm {
  gap: 8px;
}
.q-gap-md {
  gap: 16px;
}
.q-gap-lg {
  gap:  24px;
}
.q-gap-xl {
  gap:  48px;
}

**Comment by AlexDaniel** [comment]
See also https://github.com/quasarframework/quasar/discussions/16868
